ui/js/controllers/userguide.js

```diff
@@ -1,6 +1,16 @@
 "use strict";
 
-userguideApp.controller('UserguideCtrl',
-    function () {
+userguideApp.controller('UserguideCtrl', ['$scope',
+    function UserguideCtrl($scope) {
+
+        // Used for dynamic startdate, enddate param examples
+        let d = new Date();
+        let y = d.getFullYear();
+        let m = d.getMonth() + 1;
+        let sd = d.getDate() - 2;
+        let ed = d.getDate();
```

Comment:
If `sd = ed - 2` is preferable to calling `.getDate()` twice let me know and I'll switch it :)